templates/index.html

```diff
@@ -404,9 +404,9 @@ <h2>Lightweight workstation deployments</h2>
 
 <section class="p-strip--light">
   <div class="row">
-    <div class="col-5 u-vertically-center">
-      <ul class="p-logo-section">
-        <li class="p-logo-section__item">
+    <div class="col-5 u-vertically-center p-logo-section">
+      <div class="p-logo-section__items">
```

Comment:
Could we update the alignment of the logos here as well?
![image](https://user-images.githubusercontent.com/57550290/152314403-2db24196-f4ab-4c06-ade2-5e85ae30f7c4.png)

Reply:
Do you mean the spacing? This is the default spacing provided by `p-logo-section`, I'd need to add additional CSS for this. The Vanilla v3 examples are surrounded by space in the image itself, so it looks more spaced out

Reply:
I was wondering if we needed to do something because it looked different from the current page's 

![image](https://user-images.githubusercontent.com/57550290/152337932-6c6b07f2-210d-4fe7-bc59-9e9f6e8f7131.png)